cse/lang/NltkStopwordFilter.py

```diff
@@ -0,0 +1,24 @@
+from cse.lang.PreprocessorStep import PreprocessorStep
+
+class NltkStopwordFilter(PreprocessorStep):
+
+    __stopwordlist = set()
+
+
+    def __init__(self, stopwordList):
+        # lookup is faster in a set than a list
+        self.__stopwordlist = set(stopwordList)
+
+
+    @staticmethod
+    def english():
+        from nltk.corpus import stopwords
+        return stopwords.words("english")
```

Comment:
Same here. Do we need to download the list for english stopwords?

Reply:
see above